app/src/main/java/xyz/whereuat/whereuat/utils/ContactUtils.java

```diff
@@ -80,6 +83,17 @@ public static InsertCommand buildInsertCommand(Context context, String name, Str
         return new InsertCommand(context, ContactEntry.TABLE_NAME, null, values);
     }
 
+    public static int generateRandomColor() {
```

Comment:
I wouldn't be against moving this into its own utility either. But I also have no issue with leaving that for a later PR when we actually figure out what we're doing with colors.